lib/Control/Monad.hm

```diff
@@ -94,5 +94,13 @@ guardF :: Boolean  -> [Boolean]
 guardF true = [true]
 guardF false = []
 
+-- | Sequentially compose two actions, discarding any value produced
+-- by the first.
+-- This can be used to end a pipeline of monadic actions with Unit:
+-- > action1 >>= action2 >>= action3 >> pure ()
+bindUnit :: forall a b. m a -> m b -> m b
+bindUnit m k = m >>= \_ -> k
+
+infixl 1 bindUnit as >>
```

Comment:
`>>` is parsed as a part of a Binary literal. I recommend [`*>`](https://hackage.haskell.org/package/base-4.15.0.0/docs/Prelude.html#v:-42--62-)